v8/payloads/gateway.ts

```diff
@@ -59,11 +59,13 @@ export type GatewayPresenceClientStatus = Partial<Record<'desktop' | 'mobile' |
  * https://discord.com/developers/docs/topics/gateway#activity-object-activity-structure
  */
 export interface GatewayActivity {
+	id?: string;
```

Comment:
```suggestion
	id: string;
```

Reply:
I marked it as optional because bots can't send it and the same interface is used for sending

Reply:
no, that was changed with #30 :eyes: